Source/WebCore/platform/Timer.h

```diff
@@ -58,7 +58,7 @@ class TimerAlignment : public CanMakeWeakPtr<TimerAlignment> {
 };
 
 class TimerBase {
-    WTF_MAKE_TZONE_ALLOCATED(TimerBase);
+    WTF_MAKE_COMPACT_TZONE_ALLOCATED(TimerBase);
```

Comment:
`TimerBase` has derived classes:
```
SecureTextTimer
DeferrableOneShotTimer
Timer
PausableIntervalTimer
ChromeClientCallbackTimer
EventLoopTimer
EventLoopTimer
```
Do we need `WTF_MAKE_COMPACT_TZONE_...` for all of them as well?

Reply:
I think we don't actually need to compact anything in this hierarchy at all, per your other comment, so I've just dropped making TimerBase compact in the latest push.